### DISCUSSION TITLE
The lighthouse deck is always green and red

### DISCUSSION THREAD

**Discussion by jackyYu416** [bitcraze/discussions#384] [Q&A]
The lighthouse deck is always on green and red lights and updating the firmware doesn't fix the problem
![S__25624579](https://user-images.githubusercontent.com/117829285/200834096-b70722f0-e140-4dda-b97a-c0514fa4ff4d.jpg)
![1667997981126](https://user-images.githubusercontent.com/117829285/200834105-11a01273-0d3f-4c33-b0bd-82a99e23e4ef.jpg)

**Comment by ataffanel** [comment]
Looks like something is going wrong with the communication between the Crazyflie and the deck.

The most likely is that the lighthouse deck is damaged.Do you have any other lighthouse deck to test? Testing with any other deck that is using the I2C bus would also be useful to determine if the problem only comes from the lighthouse or if the Crazyflie is causing the problem.

In any way, please drop us a mail at support _at_ bitcraze.io so that we can fix a replacement.
